### PR TITLE
Feature/Add range component in filters,  update filter component

### PR DESCRIPTION
**Pull request by jszczech**

### CODE REVIEW

File: components/02-elements/banner/Banner.vue [hunk 1]
```diff
@@ -12,9 +12,9 @@
         :media="source.mediaQuery"
       >
       <img
-        class="lazyload"
         :src="image.defaultSrc"
         :alt="image.alt"
+        class="lazyload"
```
Comment (szafran89) [inline components/02-elements/banner/Banner.vue]: IMO we don't need this class at all

File: components/02-elements/dropdown-list/DropdownList.scss [hunk 2]
```diff
@@ -100,7 +98,7 @@ $dropdown-list__content-margin--screen-m  : $spacer $spacer--medium !default;
 
         &[aria-expanded="true"] {
           & > .dropdown-list__icon {
-            fill: $dropdown-list__icon-fill---secondary-open;
+            fill: $black;
```
Comment (szafran89) [inline components/02-elements/dropdown-list/DropdownList.scss]: moe to local variable

File: components/02-elements/dropdown-list/DropdownList.scss [hunk 3]
```diff
@@ -242,7 +240,7 @@ $dropdown-list__content-margin--screen-m  : $spacer $spacer--medium !default;
       background-color: $dropdown-list__item-bg-color--open;
 
       & > .dropdown-list__icon {
-        fill: $dropdown-list__icon-fill--open;
+        fill: $black;
```
Comment (szafran89) [inline components/02-elements/dropdown-list/DropdownList.scss]: move to local variable

File: components/03-modules/active-filters/ActiveFilters.vue [hunk 4]
```diff
@@ -11,27 +11,28 @@
...
-            class="active-filters__remove-icon"
             :title="closeIconTitle"
             :icon="closeIcon"
+            class="active-filters__remove-icon"
           />
         </component>
 
         <span class="active-filters__label">
-          {{ item.label }} :&nbsp;
+          {{ item.attribute_code && nameOfFilter(item.attribute_code) }} :&nbsp;
```
Comment (szafran89) [inline components/03-modules/active-filters/ActiveFilters.vue]: do we need space before `:`?

File: components/03-modules/active-filters/mocks/active-filters.json [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
     "href": "#"
   },
   {
-    "label": "Label3",
-    "value": "Value3",
+    "attribute_code": "Label3",
+    "label": "Value3",
     "ariaLabel": "Disable filter",
     "href": "#"
   }
```
Comment (szafran89) [inline components/03-modules/active-filters/mocks/active-filters.json]: move to global mocks in root of project

File: components/03-modules/filter/Filter.stories.js [hunk 6]
```diff
@@ -65,9 +65,13 @@ storiesOf('Modules/Filter', module)
...
-            :items="checkboxItems"
+            :items="checkboxItems[0].items"
+            @select="select"
           />
         </alpaca-dropdown-list>
       </app>
-    `
+    `,
+    methods: {
+      select: action('Selected'),
```
Comment (szafran89) [inline components/03-modules/filter/Filter.stories.js]: comma to remove?

File: components/03-modules/filter/color-filter/ColorFilter.js [hunk 7]
```diff
@@ -21,8 +21,8 @@ export default {
     }
   },
   methods: {
-    click(val) {
-      this.$emit('click', val)
+    click(id, label) {
+      this.$emit('click', {id: id, label: label})
```
Comment (szafran89) [inline components/03-modules/filter/color-filter/ColorFilter.js]: shorthand {id, label}

File: components/03-modules/filter/mocks/checkboxItems.json [hunk 8]
```diff
@@ -1,17 +1,42 @@
 [
   {
-    "id": "checkbox1",
-    "name": "checkbox-name1",
-    "value": "checkbox-value1"
+    "name": "Checkbox",
```
Comment (szafran89) [inline components/03-modules/filter/mocks/checkboxItems.json]: move to global mocks

File: components/03-modules/filters/Filters.js [hunk 9]
```diff
@@ -1,8 +1,10 @@
+import uniqueId from 'lodash.uniqueid'
 import AlpacaHeading from '@alpaca-storybook/components/01-globals/heading/Heading.vue'
 import AlpacaIcon from '@alpaca-storybook/components/01-globals/icon/Icon.vue'
 import AlpacaDivider from '@alpaca-storybook/components/02-elements/divider/Divider.vue'
 import AlpacaDropdownList from '@alpaca-storybook/components/02-elements/dropdown-list/DropdownList.vue'
 import AlpacaDropdownListItem from '@alpaca-storybook/components/02-elements/dropdown-list/dropdown-list-item/DropdownListItem.vue'
+import AlpacaRange from '../../03-modules/range/Range.vue'
```
Comment (szafran89) [inline components/03-modules/filters/Filters.js]: use alias `@alpaca-storybook`

File: components/03-modules/filters/Filters.js [hunk 10]
```diff
@@ -1,8 +1,10 @@
+import uniqueId from 'lodash.uniqueid'
```
Comment (szafran89) [inline components/03-modules/filters/Filters.js]: do we need uniqueId?

File: components/03-modules/range/Range.html [hunk 11]
```diff
@@ -14,53 +14,58 @@
...
+        class="input__field range-filter__input"
+        :label="upperInput.label"
+        type="text"
+        :name="upperInput.field.name"
+        :placeholder="upperInput.field.placeholder"
+      >
+      <span
+        v-if="upperInput.suffix"
+        class="range-filter__suffix"
+      >
```
Comment (szafran89) [inline components/03-modules/range/Range.html]: formatting

File: components/04-views/catalog/Catalog.scss [hunk 12]
```diff
@@ -1,5 +1,5 @@
 $catalog__margin                      : $spacer--medium 0 0 0 !default;
-$catalog__sidebar-width--mq-medium    : 306px !default;
+$catalog__sidebar-width--mq-medium    : 390px !default;
```
Comment (szafran89) [inline components/04-views/catalog/Catalog.scss]: we should use syntax with @

**Review by szafran89 — CHANGES_REQUESTED** [review]

File: components/02-elements/banner/Banner.vue (hunk 1 above)
Comment (jszczech) [reply components/02-elements/banner/Banner.vue]: 👍

File: components/02-elements/dropdown-list/DropdownList.scss (hunk 2 above)
Comment (jszczech) [reply components/02-elements/dropdown-list/DropdownList.scss]: 👍

File: components/02-elements/dropdown-list/DropdownList.scss (hunk 3 above)
Comment (jszczech) [reply components/02-elements/dropdown-list/DropdownList.scss]: 👍

File: components/03-modules/active-filters/ActiveFilters.vue (hunk 4 above)
Comment (jszczech) [reply components/03-modules/active-filters/ActiveFilters.vue]: I remove it :)

File: components/03-modules/active-filters/mocks/active-filters.json (hunk 5 above)
Comment (jszczech) [reply components/03-modules/active-filters/mocks/active-filters.json]: 👍

File: components/03-modules/filter/mocks/checkboxItems.json (hunk 8 above)
Comment (jszczech) [reply components/03-modules/filter/mocks/checkboxItems.json]: 👍

File: components/03-modules/filters/Filters.js (hunk 9 above)
Comment (jszczech) [reply components/03-modules/filters/Filters.js]: 👍

File: components/03-modules/filters/Filters.js (hunk 10 above)
Comment (jszczech) [reply components/03-modules/filters/Filters.js]: I remove it, after changes we don't need it.

File: components/03-modules/range/Range.html (hunk 11 above)
Comment (jszczech) [reply components/03-modules/range/Range.html]: 👍

File: components/04-views/catalog/Catalog.scss (hunk 12 above)
Comment (jszczech) [reply components/04-views/catalog/Catalog.scss]: 👍

**Pull request closed without merging**